mapper/tedge_mapper/src/sm_c8y_mapper/topic.rs

```diff
@@ -63,14 +67,17 @@ impl OutgoingTopic {
             Self::SoftwareListRequest => r#"tedge/commands/req/software/list"#,
             Self::SoftwareUpdateRequest => r#"tedge/commands/req/software/update"#,
             Self::SmartRestResponse => r#"c8y/s/us"#,
+            Self::RestartRequest => r#"tedge/commands/req/control/restart"#,
         }
     }
 
+    // TODO: test
```

Comment:
I don't think that a test has some value here.

* Such a test will simply be a clone of the code and will prevent no bugs, since the *actual* topic names are not those listed here.
* The best approach is to *share* these definitions between the mapper and the agent. i.e. these topics should be defined along the message structs.